Cargo.toml

```diff
@@ -133,6 +133,11 @@ path = "examples/3d/z_sort_debug.rs"
 name = "custom_loop"
 path = "examples/app/custom_loop.rs"
 
+[[example]]
+name = "draganddrop"
```

Comment:
Can we rename the example / file to "drag_and_drop"? I like to use snake_case for all file names.